Tests/YCoreUITests/Protocols/SystemImageTests.swift

```diff
@@ -88,4 +94,12 @@ extension SystemImageTests {
 
         static var textStyle: UIFont.TextStyle? { .title1 }
     }
+
+    enum RenderMode: String, CaseIterable, SystemImage {
+        case plusIcon = "plus"
+        case minusIcon = "minus"
+        case trashIcon = "trash"
```

Comment:
```suggestion
        case plus
        case minus
        case trash
```
Might as well keep it simple.